posthog/hogql_queries/insights/funnels/test/test_funnel_trends.py

```diff
@@ -2342,6 +2342,205 @@ def test_excluded_completion(self):
         self.assertEqual(0, results[0]["reached_from_step_count"])
         self.assertEqual(0, results[0]["reached_to_step_count"])
 
+    def test_breakdown_with_attribution(self):
```

Comment:
I don't fully understand how this test differs from `test_breakdown_step_attributions`. Is there a better name maybe?

Reply:
It doesn't really - one is for trends and one is for steps.

I changed the name because while I was writing them I only wanted to run one of them, and because of the complicated structure of funnel tests, it was easiest to just give the test a unique name. :-p Can make it more clear

Reply:
Ahh - in this case there is a different test file for each visualization type: `test_funnel_trends.py` (this file) for trends) and `test_funnel.py` for steps.

Reply:
Oh I see I have a test_breakdown(s)_step_attributions in both trends and steps. Let me rename these and make them more clear what they're testing.